src/clocksi_static_tx_coord_sup.erl

```diff
@@ -28,7 +28,7 @@
 
 -export([init/1]).
 
--define(NUM, 100).
+-define(NUM, 1).
```

Comment:
Why?  

This removes all of the pooling work I did to reduce contention on the supervisor and remove long message queues.